### DISCUSSION TITLE
Elastic fleet config Custom FQDN does   not apply to logstash

### DISCUSSION THREAD

**Discussion by networklord** [Security-Onion-Solutions/securityonion#13748] [2.4]
### Version

2.4.90

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Per documentation :

https://docs.securityonion.net/en/2.4/elastic-fleet.html when you add custom URL it should also apply to logstash. But only changes Fleet server Hosts

![image](https://github.com/user-attachments/assets/4db5ef75-ef5d-491e-8a7b-65ad1a990156)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by Wilkins00** [comment]
hi there,
Go to the manager and enable advance mode, go to fleet option and add  the Logstash name server, then after 10 minute it should apply the FQDN.
![image](https://github.com/user-attachments/assets/a5221f20-78bb-4fdf-98a1-6891ce58ef27)

**Reply by networklord** [reply]
Unfortunately  it does not show up :

![image](https://github.com/user-attachments/assets/70a8fcab-a17c-43b1-bc7f-f1bd8fd861d1)

**Comment by Wilkins00** [comment]
Remember it do not show up automatically, it takes around 10 to 30 to show up, try to restart the service, also press the sync button.

**Comment by Wilkins00** [comment]
I see that you are putting an Ip not a name, it might be the issue, try to add it the hosts files if a DNS entry is not created.

**Reply by networklord** [reply]
I have tried to add some DNS name but still does not show under Outputs

**Comment by Wilkins00** [comment]
Hello,
in order to add the output entry for Logstash the configuration below have to be set to false, then all the logs would be redirected to the dns added and the manager would reject the Logstash log, after a few minutes you should see the entry added.
![image](https://github.com/user-attachments/assets/cdc299d4-072a-4f02-a43f-8c29bd832eb1)

**Reply by networklord** [reply]
Hi,

Thanks for answer, i will proceed with Standalone Fleet server, dont wanna mess with options
